specs-go/config.go

```diff
@@ -58,6 +58,8 @@ type Process struct {
 	OOMScoreAdj *int `json:"oomScoreAdj,omitempty" platform:"linux"`
 	// SelinuxLabel specifies the selinux context that the container process is run as.
 	SelinuxLabel string `json:"selinuxLabel,omitempty" platform:"linux"`
+	// AllowSpeculation disables spectre mitigations
```

Comment:
could we just make it more generic like using `speculative execution` instead of `spectre`?

Reply:
I agree with you. I changed the above and config.md and the commit message.

Reply:
thanks!